esp32-controller/src/main.h

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file main.h
+ * @brief Main header file.
+ */
+
+#ifndef MAIN_H
+#define MAIN_H
+
+// Framework Includes //
+#include <Arduino.h>
+#include <Ultrasonic.h>
+#include <WiFi.h>
+#include <esp_websocket_client.h>
+
+// External Includes //
+// #include <ESPAsyncWebServer.h>
+
+// Internal Includes //
+#include "env.h"
+#include "handler/wsEventHandler.h"
+#include "modules/EngineController/EngineController.hpp"
+#include "tasks/taskTelemetry/taskTelemetry.h"
+#include "utils/typeConversion.h"
+
+// Define Constants //
+#define MESSAGE_TYPE_FORWARD 'f'
+#define MESSAGE_TYPE_BACKWARD 'b'
```

Comment:
Maybe we should define the outbound message types also. To be able to identify them in the server.

Reply:
What do you mean? Elaborate, please.

Reply:
```suggestion
// Define Constants //

// Inbound //
#define MESSAGE_TYPE_FORWARD 'f'
#define MESSAGE_TYPE_BACKWARD 'b'

// Outbound //
#define MESSAGE_TYPE_TELEMETRY 't'
```

Reply:
This comment is related to defining the message type for telemetry